### ISSUE TITLE
Can I pass in different 'thresholds' when invoking .predict() for Multilabel classification? 

### ISSUE THREAD

**Issue by aamin21** [ThilinaRajapakse/simpletransformers#252]
This is my code to try different threshold values while predicting:

<pre><code>import numpy as np
from simpletransformers.classification import MultiLabelClassificationModel

model = MultiLabelClassificationModel('distilbert', '../model_artifacts/bert', num_labels=158, use_cuda=False, args={"no_cache": True, 'threshold': 0.8, 'train_batch_size':2, 'gradient_accumulation_steps':16, 'learning_rate': 3e-5, 'num_train_epochs': 1, 'max_seq_length': 512})
r = "{My Text Here}"
predictions, raw_output = model.predict([r])

print(predicted_labels(raw_output[0], multilabel_binarizer.classes_))</code></pre>

No matter what threshold values I give , I get the same result back (11 predicted labels for my case).  I thought I would get different number of results given different thresholds. 

Looking at the code, I see (multi_label_classification_model.py, line: 115): 
`self.args = {
            "threshold": 0.5,
            "sliding_window": False,
            "tie_value": 1,
            "stride": False,
 }`

It seems the self.args.threshold is being hardcoded to 0.5 and does not support dynamic values. But, I also might have missed something.

**Comment by aamin21** [comment]
Resolution: The code is correct. The error was in my part.

**Comment by lececefifi** [comment]
hi - i have the same issue, getting the same results, and would love to know how is this resolved.

I have tried setting it in the model.predict(), 
model_args = {
       "threshold": 0.70
}

I have also added the same when model is loaded,
model = MultiLabelClassificationModel(
        model_type,
        model_name,
        args = model_args,
        use_cuda=cuda_available,

can someone point me to the right direction as where I have missed?  Thank you!
